### ISSUE TITLE
Use default CacheDir value if its missing in /etc/pacman.conf

### ISSUE THREAD

**Issue by ysoid** [pbrisbin/downgrade#47]
btw its commented in default /etc/pacman.conf

**Comment by pbrisbin** [comment]
Can you clarify a little bit? Is this a bug report that we're not finding the cache-dir when it's commented, or a feature request that we default to something when it's missing entirely?

**Comment by ysoid** [comment]
ok. It runs "find 'zero' some.pkg.xz" and it does it in $HOME in most cases. Then searching takes a lot of time and finds nothing shorely. I think it's bug )

**Comment by pbrisbin** [comment]
Can you tell me the output of the following command on your system?

```sh
sed '/^#\?CacheDir *= *\(.*\)$/!d;s//\1/' /etc/pacman.conf
```

And this, simpler one:

```sh
grep CacheDir /etc/pacman.conf
```

Thanks.

**Comment by ysoid** [comment]
Lets i show you my working conf:

[options]
UseDelta
HoldPkg     = pacman glibc
Architecture = auto
Color
SigLevel = Never
Include = /etc/pacman.d/ignore-pkg.conf
Include = /etc/pacman.d/no-upgrade.conf
Include = /etc/pacman.d/no-extract.conf
[core]
Include = /etc/pacman.d/mirrorlist
[extra]
Include = /etc/pacman.d/mirrorlist
[community]
Include = /etc/pacman.d/mirrorlist
[archlinuxfr] 
Server = http://repo.archlinux.fr/$arch

  See? This is all but it works. Here's no CacheDir to search

**Comment by pbrisbin** [comment]
Sorry, can you actually run those commands too? I'm trying to understand where `zero` could be coming from.

The code was never intended to guess at a default if it couldn't find the commented value it expects, but `find 'zero' ...` is pretty unexpected IMO.

**Comment by ysoid** [comment]
'zero' mins "", "nothing", "nihil", "0", "null", "ничего". it's just metaphor

**Comment by pbrisbin** [comment]
That makes more sense. Probably best to avoid metaphors when reporting Issues and stick to literal command examples and/or output (preferably in proper markdown syntax) to avoid this kind of confusion.

I think I understand the issue now. I would recommend adding back the commented default as a workaround for now; I'm not sure when I'll be able to add support for it being missing.

**Comment by pbrisbin** [comment]
Attempted a fix in #54, would you be willing to test that branch?
